retro/tables/pexp_5d.py

```diff
@@ -86,6 +86,9 @@ class StepSpacing(enum.IntEnum):
 PEGLEG_BREAK_COUNTER = 100
 """After how many steps without improving the llh to exit the pegleg loop adding more track segments"""
 
+MAX_CASCADE_ENEGRY = 10000000.
```

Comment:
spelling, reset to value we used in low-en (but can add comment for value you used for HESE, and we'll figure out something more sophisticated later to do with this)